mkosi

```diff
@@ -1438,6 +1438,17 @@ def install_extra_trees(args, workspace, for_cache):
         for d in args.extra_trees:
             enumerate_and_copy(d, os.path.join(workspace, "root"))
 
+def install_skeleton_trees(args, workspace, for_cache):
+    if args.skeleton_trees is None:
+        return
+
+    if for_cache:
+        return
+
+    with complete_step('Copying in skeleton file trees'):
+        for d in args.skeleton_trees:
+            enumerate_and_copy(d, os.path.join(workspace, "root"))
```

Comment:
now that I merged @keszybz copy() rework, could you update this bit accordingly?